cms/djangoapps/contentstore/utils.py

```diff
@@ -160,7 +160,9 @@ def get_lms_link_for_certificate_web_view(user_id, course_key, mode):
 
 def course_image_url(course):
     """Returns the image url for the course."""
-    loc = StaticContent.compute_location(course.location.course_key, course.course_image)
+    loc = u''
+    if course.course_image:
+        loc = StaticContent.compute_location(course.location.course_key, course.course_image)
```

Comment:
@awaisdar001 I would find the following preferable since it would handle all cases of invalid `course_image` values. But from your earlier comment, it seems there are some limitations of this?
```
try:
  loc = StaticContent.compute_location(course.location.course_key, course.course_image)
except InvalidKeyError:
  loc = ''
```